samples/bluetooth/fast_pair/locator_tag/configuration/pm_static_nrf54l15dk_nrf54l10_cpuapp.yml

```
@@ -0,0 +1,60 @@
+mcuboot:
+  address: 0x0
+  region: flash_primary
+  size: 0x5000
+
+mcuboot_primary:
+  address: 0x5000
+  orig_span: &id001
+  - app
+  - mcuboot_pad
+  region: flash_primary
+  size: 0x79000
+  span: *id001
+mcuboot_pad:
+  address: 0x5000
+  region: flash_primary
+  size: 0x800
+app:
+  address: 0x5800
+  region: flash_primary
+  size: 0x78800
+mcuboot_primary_app:
+  address: 0x5800
+  orig_span: &id002
+  - app
+  region: flash_primary
+  size: 0x78800
+  span: *id002
+
+mcuboot_secondary:
+  address: 0x7e000
+  orig_span: &id003
+  - mcuboot_secondary_pad
+  - mcuboot_secondary_app
+  region: flash_primary
+  size: 0x79000
+  span: *id003
+mcuboot_secondary_pad:
+  region: flash_primary
+  address: 0x7e000
+  size: 0x800
+mcuboot_secondary_app:
+  region: flash_primary
+  address: 0x7e800
+  size: 0x78800
+
+bt_fast_pair:
+  address: 0xf7000
+  region: flash_primary
+  size: 0x1000
+
+settings_storage:
+  address: 0xf8000
+  region: flash_primary
+  size: 0x7000
+
+unused:
+  address: 0xff000
+  region: flash_primary
+  size: 0x800
```

strongly consider same approach that is used in nrf_desktop, so use remaining 0x800 for bt_fast_pair partition and creating bigger settings partition :D bt_fast_pair partition needs only size of 0x48 so it would be perfect to utilize stuff that can't be used for settings :D 
```suggestion
settings_storage:
  address: 0xf7000
  region: flash_primary
  size: 0x8000

bt_fast_pair:
  address: 0xff000
  region: flash_primary
  size: 0x800
```

---

I decided to make settings storage smaller than the one on nRF54L15 SoC and increase the application partitions

---

improved the memory map, thanks for this suggestion!